packages/react-router-dom-v5-compat/__tests__/compat-router-test.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * @jest-environment node
+ */
+import * as React from "react";
+import * as ReactDOMServer from "react-dom/server";
+import { act } from "react-dom/test-utils";
+import { CompatRouter, Routes } from "../index";
+
+// Have to mock react-router-dom to have a comparable API to v5, otherwise it will
+// be using v6's API and fail
+jest.mock("react-router-dom", () => ({
+  useHistory: () => ({ location: "/" }),
+}));
```

Comment:
Don't super love having to mock this out, but without this mock `CompatRouter` was trying to use `useHistory` from the `react-router-dom` version it had (v6, where it doesn't exist). Since this test is pretty simple I figured this would be fine, but let me know otherwise.